tasks/bash.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
```

Comment:
this should be tested a bit, I don't know if this works out of the box on all operating systems